rules/gdi-aircraft.yaml

```diff
@@ -862,3 +862,45 @@ KODIAK:
 	WithVoxelBody:
 	WithMuzzleOverlay:
 	HitShape:
+
+Hellfire_beacon:
+	Inherits: ^AttackPlane
+	Inherits@FeatureTargettype: ^FeatureTargettypeFloraAir
```

Comment:
Then you should declare the dependency of this PR. In the current state, it crashes for me.